src/main/java/uk/gov/hmcts/reform/roleassignment/util/ValidationUtil.java

```diff
@@ -68,6 +68,35 @@ public static boolean validateTextHyphenField(String field) {
         return (field != null);
     }
 
+    public static boolean validateDateTime(String strDate) {
+
+        if (strDate.length() < 16) {
+            return false;
+        }
+        SimpleDateFormat sdfrmt = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm");
+        sdfrmt.setLenient(false);
+        try {
+            Date javaDate = sdfrmt.parse(strDate);
+            LOG.info("TTL {}", javaDate);
+        } catch (ParseException e) {
+            return false;
+        }
+        return true;
+    }
+
+    public static boolean validateDateOrder(String beginTime, String endTime) throws ParseException {
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm");
+        Date beginTimeP = sdf.parse(beginTime);
+        Date endTimeP = sdf.parse(endTime);
+        boolean result = false;
+        if (endTimeP.after(beginTimeP)) {
```

Comment:
we need to add few more checks for both begin and endtime to be greater than current time.